pkg/products/cloudresources/reconciler.go

```diff
@@ -357,6 +357,10 @@ func (r *Reconciler) reconcileCIDRValue(ctx context.Context, client k8sclient.Cl
 		return err
 	}
 
+	if cidrValue == "" {
+		cidrValue = "10.1.0.0/16"
+	}
+
 	if !ok {
 		return nil
 	}
```

Comment:
```suggestion
		//!ok means the param wasn't found so we want to default rather than return
	if !ok || cidrValue == "" {
		cidrValue = "10.1.0.0/16"
	}
```